src/nisarqa/parameters/nisar_params.py

```diff
@@ -1039,15 +1041,39 @@ class ProductPathGroupParamGroup(YamlParamGroup):
     ----------
     qa_output_dir : path-like, optional
         Filepath to the output directory to store NISAR QA output files.
+        If the directory does not exist, it will be created.
         Defaults to './qa'
+    scratch_dir : path-like, optional
+        Directory where QA software may write temporary data.
+        If the directory does not exist, it will be created.
+        Because this scratch directory might be shared with e.g. ISCE3
+        science product SASes, QA will create a uniquely-named
+        directory inside `scratch_path` for any QA scratch files.
+        Provided `scratch_dir` argument will be converted to a Path object.
+        Defaults to './scratch'
     """
 
     qa_output_dir: str | os.PathLike = field(
         default="./qa",
         metadata={
             "yaml_attrs": YamlAttrs(
                 name="qa_output_dir",
-                descr="""Output directory to store all QA output files.""",
+                descr="""Output directory to store all QA output files.
+                If the directory does not exist, it will be created.""",
+            )
+        },
+    )
+
+    scratch_dir: str | os.PathLike = field(
+        default="./scratch",
+        metadata={
+            "yaml_attrs": YamlAttrs(
+                name="scratch_path",
+                descr="""Directory where QA software may write temporary data.
```

Comment:
Not sure we should specify "QA" in the help message here, since the SAS may also use this directory.

Here's the help string I used for the static layers runconfig:

https://github-fn.jpl.nasa.gov/gunter/isce/blob/ab362769e956a5464735d45682223461fa01374e/share/nisar/defaults/static.yaml#L58-L60